src/components/Modals/UploadDocumentModal.jsx

```diff
@@ -97,7 +102,11 @@ const UploadDocumentModal = ({ showModal, setShowModal }) => {
   return (
     <Dialog open={showModal} aria-labelledby="upload-document-dialog" onClose={clearInputFields}>
       <FormSection title="Upload Document">
-        <form onSubmit={handleDocUpload} autoComplete="off">
+        <form
+          onSubmit={handleDocUpload}
+          autoComplete="off"
+          style={{ width: isSmallScreen ? '100%' : '100%' }}
```

Comment:
No need for ternary

Reply:
Think we can remove this entirely. Just tested it without having much effect on the styling desktop or mobile.